papas/graphtools/src/EventRuler.cpp

```diff
@@ -11,18 +11,18 @@ Distance EventRuler::distance(Identifier id1, Identifier id2) const {
   // figure out the object types and then call ClusterCluster or ClusterTrack distance measures
   if (IdCoder::isCluster(id1) && IdCoder::isCluster(id2))
     if (IdCoder::type(id1) == IdCoder::type(id2))
-      return std::move(clusterClusterDistance(id1, id2));
+      return clusterClusterDistance(id1, id2);
```

Comment:
What is the problem with std::move? seems to me that now, we're copying the object (if Distance is an object)